Content.Server/Lathe/LatheSystem.cs

```diff
@@ -200,6 +203,15 @@ private void OnGetRecipes(EntityUid uid, TechnologyDatabaseComponent component,
                     continue;
                 args.Recipes.Add(recipe);
             }
+
+            foreach (var recipe in latheComponent.EmagRecipes)
+            {
+                if (!HasComp<EmaggedComponent>(uid))
+                    continue;
```

Comment:
this condition should just be checked outside of the loop with a return statement.